vnext/Microsoft.ReactNative/Views/ShadowNodeBase.h

```diff
@@ -107,6 +107,10 @@ struct REACTWINDOWS_EXPORT ShadowNodeBase : public facebook::react::ShadowNode {
   comp::CompositionPropertySet EnsureTransformPS();
   void UpdateTransformPS();
 
+  bool IsRegisteredForMouseEvents() const {
+    return m_onMouseEnter || m_onMouseLeave || m_onMouseMove;
```

Comment:
I know you aren't adding these fields but consider renaming them since right now they would appear to be events/delegates instead of representing input state.
Maybe these should be inside an InputState class and you can have a member of that type

Reply:
renamed.